### PR TITLE
cairomm 1.11.2

### PR DESCRIPTION
**Pull request by dunn**
Too big a test?

### CODE REVIEW

File: Library/Formula/cairomm.rb [hunk 1]
```diff
@@ -17,21 +14,59 @@ class Cairomm < Formula
...
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cc").write <<-EOS.undent
+      #include <string>
+      #include <iostream>
+      #include <cmath>
```
Comment (tdsmith) [inline Library/Formula/cairomm.rb]: Are you using iostream or cmath?

File: Library/Formula/cairomm.rb [hunk 2]
```diff
@@ -17,21 +14,59 @@ class Cairomm < Formula
...
+      -I#{lib}/cairomm-1.0/include
+      -I#{include}/cairomm-1.0
+      -I#{Formula["cairo"].opt_include}/cairo
+      -I#{Formula["freetype"].opt_include}/freetype2
+      -I#{Formula["libsigc++"].opt_include}/sigc++-2.0
+      -I#{Formula["libsigc++"].opt_lib}/sigc++-2.0/include
+      -L#{lib}
+      -lcairomm-1.0
+    ]
+    system ENV.cxx, *flags, "test.cc", "-o", "test"
```
Comment (tdsmith) [inline Library/Formula/cairomm.rb]: This syntax doesn't work in Ruby 1.8: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24249/version=mountain_lion/console

`*args` have to be at the end of the argument list. You can do `*(flags + ["some", "other", "things"])` but it's a little ugly; may as well just push everything into flags?

**Comment by tdsmith** [comment]
Test looks great to me :+1:

File: Library/Formula/cairomm.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/cairomm.rb]: Nope.  I thought it was failing to build without them but it was an unrelated error.

**Comment by jacknagel** [comment]
cairomm 1.11 is a development release. We want to stick with stable releases.

**Comment by jacknagel** [comment]
for reference: http://cgit.freedesktop.org/cairomm/commit/?id=6321b86d98c44e0996b457e136476694ec3ec2f2

**Comment by dunn** [comment]
Gotcha.

**Comment by tdsmith** [comment]
I think this is a stable release -- even micros are releases, not even minors. Their release directory only has even micros, which is consistent, and identifies 1.11.2 as LATEST: http://www.cairographics.org/releases/

**Comment by jacknagel** [comment]
In the NEWS file it is listed as "unstable".

**Pull request closed without merging**